core/chains/evm/config/toml/defaults.go

```diff
@@ -33,48 +33,83 @@ var (
 )
 
 func init() {
-	// read the defaults first
+	// read all default configs
+	initReadDefaults()
+
+	// overrides should be applied as:
+	// fallback.toml(defaults dir) <- fallback.toml(env CL_CHAIN_FALLBACK) <- ChainSpecific.toml(env CL_CHAIN_DEFAULTS)
+	//
+	// the custom fallback gets processed and overrides the default fallback
+	if path := env.CustomFallback.Get(); path != "" {
```

Comment:
can we lock this in with a testscripts test like in https://github.com/smartcontractkit/chainlink/pull/14090/files#diff-ea10429fdb1fbc100e3d8082e39e6d40a12f82b48c1f37ab9158b6c3c0e44d81 ?

Reply:
Good idea. I'll add that